core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -99,6 +100,30 @@ public function load(array $configs, ContainerBuilder $container): void
         ;
     }
 
+    /**
+     * Validates and sets the "contao.image.sizes" parameter.
+     */
+    private function setPredefinedImageSizes(array $config, ContainerBuilder $container): void
+    {
+        if (!isset($config['image']['sizes']) || count($config['image']['sizes']) === 0) {
```

Comment:
CS: `0 === \count($config['image']['sizes'])`

You should probably run the CS fixer. 😄